app/preload.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
This is a cool pattern! Not the slickest (having to define a global var and all) but I see why it's necessary.

Reply:
I do hope to make this file `.ts` soon with it's own webpack config.